pr_check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="ingress"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="ingress"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```

Comment:
We might be able to dump payload-tracker-go in here. I guess it depends on the resourceTemplate name we want to use in app-interface.